net/fetcher_test.go

```diff
@@ -0,0 +1,77 @@
+package net_test
+
+import (
+	"context"
+	"testing"
+
+	"gx/ipfs/QmPVkJMTeRC6iBByPWdrRkD3BE5UXsj5HPzb4kPqL186mS/testify/require"
+	"gx/ipfs/QmR8BauakNcBa3RbE4nbQu76PDiJgoQgz8AJdhJuiU4TAw/go-cid"
+	ipld "gx/ipfs/QmRL22E4paat7ky7vx9MLpR97JHHbFPrg3ytFQw6qp1y1s/go-ipld-format"
+	bstore "gx/ipfs/QmRu7tiRnFk9mMPpVECQTBQJqXtmG132jJxA1w9A7TtpBz/go-ipfs-blockstore"
+	"gx/ipfs/QmSz8kAe2JCKp2dWSG8gHSWnwSmne8YfRXTeK5HBmc9L7t/go-ipfs-exchange-offline"
+	"gx/ipfs/QmUadX5EcvrBmxAV9sE7wUWtWSqxns5K84qKJBixmcT1w9/go-datastore"
+	dss "gx/ipfs/QmUadX5EcvrBmxAV9sE7wUWtWSqxns5K84qKJBixmcT1w9/go-datastore/sync"
+	bserv "gx/ipfs/QmZsGVGCqMCNzHLNMB6q4F6yyvomqf1VxwhJwSfgo1NGaF/go-blockservice"
+
+	"github.com/filecoin-project/go-filecoin/net"
+	"github.com/filecoin-project/go-filecoin/types"
+)
+
+func requireBlockStorePut(require *require.Assertions, bs bstore.Blockstore, data ipld.Node) {
+	err := bs.Put(data)
+	require.NoError(err)
+}
+
+func TestFetchHappyPath(t *testing.T) {
```

Comment:
blocking: this is a good test but iirc does it actually test round trip? it doesn't test that two separate services connected to each other can successfully pull from one another, does it?

Reply:
Thanks I wasn't aware of this convention.

Reply:
I want to make sure that the offending terminology is `rtBlocks` and `rtCids`.  If it's something else please advise and I'll fix that too.